influunt-api/modules/influunt-api/app/controllers/api/ControladoresController.java

```diff
@@ -542,6 +577,11 @@
             return CompletableFuture.completedFuture(status(UNPROCESSABLE_ENTITY, Json.toJson(erros)));
         } else {
             if (controladorJaExiste) {
+                if(stepAneis){
+                    Controlador controlador_anterior = Controlador.find.byId(controlador.getId());
+                    aneisAnterior = (int) controlador_anterior.getAneis().stream().filter(Anel::isAtivo).count();
+                    stepAneis = false;
```

Comment:
Acho que ficou faltando a chamada para atualizar a quantidade de anéis do controlador aqui
```java
controlador.atualizarStatusControlador(StatusVersao.EM_CONFIGURACAO, StatusVersao.EM_CONFIGURACAO, 1, 1, aneisAnterior, 0);
```